cmake/projects/OpenSSL/schemes/url_sha1_openssl_ios.cmake.in

```diff
@@ -33,9 +34,26 @@ hunter_test_string_not_empty("@IOS_SDK_VERSION@")
 hunter_test_string_not_empty("@IPHONESIMULATOR_ROOT@")
 hunter_test_string_not_empty("@IPHONEOS_ROOT@")
 
+hunter_dump_cmake_flags(SKIP_INCLUDES)
+# -> CMAKE_CXX_FLAGS
+# -> CMAKE_C_FLAGS
+
 hunter_unsetvar(ssl_input_libraries)
 hunter_unsetvar(crypto_input_libraries)
 
+string(COMPARE EQUAL "${CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE}" "YES" _embed_bitcode)
```

Comment:
This logic should go to `hunter_dump_cmake_flags`. I guess we may need this for other packages too (like Boost).